includes/class-wc-admin-note.php

```diff
@@ -0,0 +1,463 @@
+<?php
+/**
+ * WooCommerce Admin (Dashboard) Notes.
+ *
+ * The WooCommerce admin notes class gets admin notes data from storage and checks validity.
+ *
+ */
+defined( 'ABSPATH' ) || exit;
+
+class WC_Admin_Note extends WC_Data {
```

Comment:
Missing docblock.

Reply:
Fixed in https://github.com/woocommerce/wc-admin/pull/464/commits/4433670cb7eb3ecf2b4d05ed34b83b6ab4e6dea8